README.md

```diff
@@ -291,6 +291,14 @@ $ lerna run build
 
 Run an [npm script](https://docs.npmjs.com/misc/scripts) in each package that contains that script.
 
+### Import
+
+```sh
+$ lerna import <path-to-external-repository>
+```
+
+Import the package at `<path-to-external-repository>`, with commit history.
```

Comment:
Added some more detail here.  I worry that using lerna-changelog as an example might be confusing, since it has "lerna" in the name, so I continued to use placeholders.

I also lower-cased the section heading to match the changes from @kaicataldo over the weekend (which I've picked up via rebase).